docs/content/contribute/index.md

```diff
@@ -3,19 +3,25 @@ title: Contribute to HyperShift
-* Always keep refactorings (how we do something) separate from logic changes (what we do).
-* Keep changes scoped to one thing and as minimal as possible. If you find additional things along the way that you feel should be improved, do that in a separate pull request. This helps ensure that you will get a timely review of your change, as a series of small pull requests is a lot easier to review than one big pull request that changes 10 independent things for independent reasons.
-* Before submitting the pull request on GitHub, look at your changes and try to view them from the eyes of a reviewer. Try to find the aspects that might not immediately make sense for someone else and explain them in the pull request description. Also, ensure that the previous point (scoped to one thing and as minimal as possible) was followed.
-* This repository is the base code for the Hypershift Operator and the Control Plane Operator (belongs to the OCP payload) so they might have different release cadence. To make it clear which component your PR impacts:
-  * Every PR must have either `area/hypershift-operator`, `area/control-plane-operator` or both labels.
-* Every PR must have at least one of these labels: `area/hypershift-operator`, `area/control-plane-operator`, `area/documentation`, `area/cli`, or `area/testing`
-* Every PR title must be prefixed with the Jira ticket that is addressing e.g. [https://github.com/openshift/hypershift/pull/2233](https://github.com/openshift/hypershift/pull/2233).
\ No newline at end of file
+## Creating a Pull Request
+1. For small changes, you can just do the change and submit a pull request with it. 
+2. For bigger changes (more than 200 lines of code diff), do not just do the change but, ask for feedback on the idea and direction of the change first (Either in a GitHub issue or the #project-hypershift channel in the External Red Hat Slack). 
+    1. This avoids situations where big changes are submitted that are then declined or never reviewed, which is frustrating for everyone. 
+3. Regardless of the size of the change, always explain how the change will improve the project.
+4. Every PR title must be prefixed with the Jira ticket that is addressing e.g. [https://github.com/openshift/hypershift/pull/2233](https://github.com/openshift/hypershift/pull/2233).
+5. This repository is the base code for the Hypershift Operator and the Control Plane Operator (belongs to the OCP payload) so they might have different release cadence. To make it clear which component your PR impacts:
+    1. Every PR must have at least one of these labels: `area/hypershift-operator`, `area/control-plane-operator`, `area/documentation`, `area/cli`, or `area/testing`
```

Comment:
This is not going to be actionable outside of the core team. The label should be automatically added anyway, so not sure how beneficial this explanation is.

Reply:
Ok I removed that part.